Doc/library/importlib.metadata.rst

```diff
@@ -320,6 +386,22 @@ function is not reliable with such installs.
 Distributions
 =============
 
+.. class:: Distribution
+
+   Details of an installed distribution package (as described below).
+
+   Note: different ``Distribution`` instances do not currently compare
+   equal, even if they relate to the same installed distribution (and
+   hence have the same attributes)
```

Comment:
```suggestion
   Details of an installed distribution package.

   Note: different ``Distribution`` instances do not compare
   equal, even if they relate to the same installed distribution and
   have the same attributes.
```

Reply:
Applied a variation on this locally (I want to keep the "currently" weasel wording here, as not being able to select on `dist` when querying entry points is mildly annoying, and making `dist` instance comparison actually work would be one way to do that. Leaving "currently" out makes this sound too much like an intentional design decision instead of an accident of implementation).